### PR TITLE
[GL-5735] Documenting aliases and the new GRN format

### PR DESCRIPTION
**Pull request by elagulsen**
Documenting the new GRN types with aliases. Lmk if I should clarify any wording!

### CODE REVIEW

File: pages/docs/data-ops/grns.mdx [hunk 1]
```diff
@@ -6,12 +6,24 @@ always remain the same.
 
 ## Anatomy of a GRN
 
-A GRN is comprised of two components, separated by `:`
+A GRN is comprised of three components: type, id, and alias, separated by `:`
```
Comment (meyer-glean) [inline pages/docs/data-ops/grns.mdx:9]: lol https://www.npr.org/2015/03/12/392568604/dont-you-dare-use-comprised-of-on-wikipedia-one-editor-will-take-it-out

(no change needed just funny)

File: pages/docs/data-ops/grns.mdx [hunk 2]
```diff
@@ -6,12 +6,24 @@ always remain the same.
...
+
+```bash
+type:id:alias
+```
+
+Not all 3 components are necessary to specify a GRN. It is also possible to specify a GRN without using its alias:
```
Comment (meyer-glean) [inline pages/docs/data-ops/grns.mdx:15]:
```suggestion
The alias can be omitted as long as an identifier is present:
```

File: pages/docs/data-ops/grns.mdx [hunk 3]
```diff
@@ -49,3 +64,17 @@ the Glean web app into DataOps, without recreating it.
 
 Anywhere you specify a filepath in a DataOps config, you may instead specify a GRN.
 This allows you to reference resources independently of how you structure your files.
+
+### With dbt
+
+When referencing models deployed through dbt, the standard GRN format is valid.
+In addition, the following format can be used to reference dbt models:
```
[the remark this answers is not in the document]
Comment (dse0) [reply pages/docs/data-ops/grns.mdx:71]: This seems fine to me?

**Review by dse0 — COMMENTED** [review]
Thoughts on renaming this page from `GRNs` to `Resource identifiers`?

**Comment by elagulsen** [comment]
@dse0 personally I think "resource identifier" is a bit of a mouthful? I've heard others suggest we change GRN to stand for "global resource name" or "general resource name" instead, which I like

**Comment by dse0** [comment]
> @dse0 personally I think "resource identifier" is a bit of a mouthful? I've heard others suggest we change GRN to stand for "global resource name" or "general resource name" instead, which I like

To clarify, I just meant the title of the documentation page, if you don't know what a GRN is yet you're unlikely to click on it here:
Screen Shot 2023-08-14 at 4 29 00 PM

Also this page is now also about aliases. So my thought is the title of the page should be about the goal of the features, identifying resources in your dataops files. Don't feel strongly though.

**Comment by elagulsen** [comment]
@dse0 Sorry, I misread your comment. Changing the name of the page is all good with me! I'll update this w/ Meyer's suggestions & the name change, & then push again

**Comment by dse0** [comment]
Looks like there's a few conflicts to resolve

**Pull request merged**